docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,177 @@
+
+Package metadata elements can also be specified directly in the project file (.csproj).
+
+Below is a table mapping and describing all the available packages metadata elements:
+
+| Visual Studio property name | csproj/MSBuild property name | Nuspec property name        | Description                                                                                                       |
+|-----------------------------|------------------------------|-----------------------------|-------------------------------------------------------------------------------------------------------------------|
+| `Package id`                | `PackageId`                  | `id`                        | The package identifier. A prefix from the identifier can be reserved if it meets the  criteria.                   |
+| `Package Version`           | `Version`                    | `version`                   | NuGet package version. For more information, see NuGet package version.                                           |
+| `Authors`                   | `Authors`                    | `authors`                   | A comma-separated list of package authors, matching the profile names on NuGet.org.                               |
+| `Description`               | `Description`                | `description`               | A long description of the package displayed in UI.                                                                |
+| `Copyright`                 | `Copyright`                  | `copyright`                 | Copyright details for the package. language.                                                                      |
+| `Licensing - Expression`    | `PackageLicenseExpression`   | `license type="expression"` | An SPDX license expression or path to a license file within the package, often shown in UIs like NuGet.org.       |
+| `Licensing - File`          | `PackageLicenseFile`         | `license type="file"`       | File for custom license that isn't supported by license expressions                                               |
+| `Project URL`               | `PackageProjectUrl`          | `projectUrl`                | A URL for the project homepage.                                                                                   |
+| `Icon File`                 | `PackageIcon`                | `icon`                      | An image to use as the icon for the package.                                                                      |
+| `Repository URL`            | `RepositoryUrl`              | `repository url=""`         | URL to the repository from which the package was built.                                                           |
+| `Repository type`           | `RespositoryType`            | `repository type=""`        | Type of repository the repository URL is pointing to (i.e. git)                                                   |
```

Comment:
I'd suggest that customers packing via msbuild (dotnet cli uses msbuild) would be better off using one of the SourceLink packages, which automatically adds the repository type and url metadata.

edit: ah, I see you're added it further down. Maybe it's ok as it is now, but also consider if customers are going to read this, stop reading, go to VS and make edits, and then come back to read more of the guidelines.